static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -307,6 +309,14 @@ CStudioAuthoring.Module.requireModule(
 
           rteStyleOverride = rteConfig.rteStyleOverride ? rteConfig.rteStyleOverride : null;
 
+          styleFormats =
+            rteConfig.styleFormats && rteConfig.styleFormats.length != 0
+            ? Function(`"use strict"; return (${rteConfig.styleFormats})`)()
+            : [];
+
+          styleFormatsMerge = rteConfig.styleFormatsMerge && rteConfig.styleFormatsMerge !== 0
+            ? rteConfig.styleFormatsMerge === 'true' : null;
```

Comment:
`rteConfig.styleFormatsMerge && rteConfig.styleFormatsMerge !== 0` looks redundant. If `rteConfig.styleFormatsMerge` is true, `rteConfig.styleFormatsMerge !== 0` will always be true too.

As said on the comment above, initialize here using whatever logic and just use below.

This would be sufficient to keep the current defaults:
```suggestion
          styleFormatsMerge = rteConfig.styleFormatsMerge === 'true';
```

Reply:
Thanks. I fixed!